omnibus-arbitrum-sepolia.toml

```
@@ -238,6 +238,14 @@ description = "1 hour"
 defaultValue = "<%= String(1 * 3600) %>"
 description = "1 hour"
 
+[setting.USDe_staleness_tolerance]
+defaultValue = "<%= String(1 * 3600) %>"
+description = "1 hour"
+
+[setting.sol_staleness_tolerance]
```

can we keep casing consistent
```suggestion
[setting.SOL_staleness_tolerance]
```